arc/database/api.py

```
@@ -92,14 +92,44 @@ async def start_simulation(
+                    scenario_dict = {"id": f"scenario_{i}", "task": str(scenario)}
+                
+                scenario_dicts.append(scenario_dict)
+                scenario_ids.append(scenario_dict.get("id", f"scenario_{i}"))
+            
+            # First, create/ensure scenarios exist in the scenarios table
+            async with self.db.engine.begin() as conn:
+                for i, scenario_dict in enumerate(scenario_dicts):
+                    scenario_id = scenario_dict.get("id", f"scenario_{i}")
+                    scenario_name = scenario_dict.get("name", scenario_dict.get("task", f"Scenario {i+1}"))
+                    task_prompt = scenario_dict.get("task", scenario_dict.get("task_prompt", ""))
+                    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)